Tasks/DeleteFilesV1/deletefiles.ts

```diff
@@ -85,6 +87,22 @@ tl.setResourcePath(path.join(__dirname, 'task.json'));
         }
     }
 
+    // if there wasn't an error, check if there's anything in the folder tree other than folders
+    // if not, delete the root as well
+    if (removeSourceFolder && !errorHappened) {
+        foundPaths = tl.find(sourceFolder);
+
+        if (foundPaths.length === 1) {
```

Comment:
Maybe helpful to debug log length in an else to make it easier to debug?

Reply:
Yeah, not a bad idea. I'll do that.